Solutions/SAP/sapcon-sentinel-kickstart.sh

```diff
@@ -44,11 +44,11 @@ The access and password must be granted explicitly to you by Microsoft, is perso
 - SAP change requests: Import any required change requests for your logs from the CR folder of this repository - https://github.com/Azure/AzureSentinel4SAP/tree/main/CR.
```

Comment:
This link https://github.com/Azure/AzureSentinel4SAP/tree/main/CR is not working, Can you please check and update. Thanks!

Reply:
@YoavDaniely : Please the link.. it is not working... Thanks!!!